Cargo.lock

```diff
@@ -6110,7 +6113,7 @@ dependencies = [
 [[package]]
 name = "pallet-author-inherent"
 version = "0.9.0"
-source = "git+https://github.com/moondance-labs/nimbus?branch=tanssi-polkadot-v0.9.40#007fb6d37c8f65c505931fccd2494276865fd99e"
+source = "git+https://github.com/moondance-labs/nimbus?branch=tanssi-polkadot-v0.9.40#24d282222894a564d0cb98217de7d9ec7bc063c9"
```

Comment:
Was this intentional? There is another commit after this: https://github.com/moondance-labs/nimbus/commits/tanssi-polkadot-v0.9.40

Reply:
yes we can point to the new commit to make sure we dont need the commit I reverted